java-frontend/src/main/java/org/sonar/java/resolve/BytecodeAnnotationVisitor.java

```diff
@@ -53,24 +52,28 @@ public AnnotationVisitor visitAnnotation(String name, String desc) {
 
   @Override
   public void visitEnum(String name, String desc, String value) {
-    List<JavaSymbol> lookup = getSymbol(desc).members().lookup(value);
-    for (JavaSymbol symbol : lookup) {
-      if (symbol.isKind(JavaSymbol.VAR)) {
-        addValue(name, symbol);
-      }
-    }
+    getSymbol(desc).members().lookup(value).stream()
+      .filter(symbol -> symbol.isKind(JavaSymbol.VAR))
+      .forEach(symbol -> addValue(name, symbol));
   }
 
   @Override
   public AnnotationVisitor visitArray(final String name) {
     final List<Object> valuesList = Lists.newArrayList();
-    //TODO handle arrays of annotation and arrays of enum values.
+    // TODO handle arrays of annotation
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&branch=SONARJAVA-1993&issues=AWEEw-oCf6JiG_HArtjv&open=AWEEw-oCf6JiG_HArtjv)

[ISSUE_KEY]: AWEEw-oCf6JiG_HArtjv

Reply:
I won't fix this remaining part, as I just improved the current handling to cover enums. Marked as "Confirmed".